web/src/components/core/fileviewer.scss

```diff
@@ -0,0 +1,7 @@
+
+.filecontent {
```

Comment:
Please, remember to move this to the _src/assets/styles/blocks.css_ to avoid using ad-hoc CSS files by now.